icu4j/maven-build/maven-icu4j/pom.xml

```diff
@@ -0,0 +1,493 @@
+<?xml version="1.0"?>
+<!--
+* © 2016 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+*******************************************************************************
+* Copyright (C) 2010-2016, International Business Machines Corporation and    *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>73.1-SNAPSHOT</version>
+    <!-- default relativePath for parent -->
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icu4j</artifactId>
+  <version>73.1-SNAPSHOT</version>
+
+  <packaging>jar</packaging>
+
+  <name>ICU4J</name>
+  <description>International Component for Unicode for Java (ICU4J) is a mature, widely used Java library providing Unicode and Globalization support</description>
+  <url>https://icu.unicode.org/</url>
+  <inceptionYear>2001</inceptionYear>
+
+  <licenses>
```

Comment:
In general you don't want to duplicate in a child everything the parent already has.
In this case there is a lot (description, licenses, developers, mailingLists, scm, issueManagement, distributionManagement.

Everything between the initial `parent` + `packaging` to `<properties>`
And probably a big chunk of the `<properties>` need to be moved to the parent (the ones that are shared between jars)

Reply:
Done. Agreed. I intentionally left this cleanup for later because I wanted to focus on ensuring that the important hard parts had a feasible good solution.